<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HMainContextOptions</class>
 <widget class="QWidget" name="HMainContextOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>49</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
	background-color: rgb(0, 0, 0);
	color: rgb(255, 255, 255);
}

QToolButton:checked {
	color: rgb(255, 255, 255);
	font: 75 18pt &quot;Candara&quot;;
}

QToolButton:!checked {
	color: rgb(125, 125, 125);
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <item>
    <widget class="QToolButton" name="toolButton_allArtists">
     <property name="font">
      <font>
       <family>Candara</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Top Artists</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="toolButton_suggestions">
     <property name="font">
      <font>
       <family>Candara</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Suggestions</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="toolButton_local">
     <property name="font">
      <font>
       <family>Candara</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Local</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QToolButton" name="toolButton_album">
     <property name="font">
      <font>
       <family>Candara</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="icons.qrc">
       <normaloff>:/icons/view_album.png</normaloff>
       <activeoff>:/icons/view_album_sel.png</activeoff>
       <selectedoff>:/icons/view_album_sel.png</selectedoff>
       <selectedon>:/icons/view_album_sel.png</selectedon>:/icons/view_album.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="toolButton_list">
     <property name="font">
      <font>
       <family>Candara</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="icons.qrc">
       <normaloff>:/icons/view_list.png</normaloff>
       <activeoff>:/icons/view_list_sel.png</activeoff>
       <selectedoff>:/icons/view_list_sel.png</selectedoff>:/icons/view_list.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="toolButton_play">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="icons.qrc">
       <normaloff>:/icons/media-play-alt.png</normaloff>:/icons/media-play-alt.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
